specification/regionmove/resource-manager/readme.md

```diff
@@ -0,0 +1,79 @@
+# regionmove
```

Comment:
why do you use a separate location for the readme files?

Reply:
Hey, when we raised this PR earlier for private preview, these files were auto-generated by HUB.
However, what is the location preferred for readme files?